### PR TITLE
Login signup popup

### PR DESCRIPTION
**Pull request by vanshitalavange**
https://user-images.githubusercontent.com/86543154/154512335-76a0d934-8da6-402e-a459-0ee53c2bcaff.mp4
- created login signup popup
- used 'required' on input fields
- Live preview: https://deploy-preview-2--the-organic-store.netlify.app/#

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,8 +11,10 @@
     />
     <link rel="icon" type="image/x-icon" href="./assets/ma_logo.png" />
     <link href="./stylesheets/pages/index.css" rel="stylesheet" />
+    <link href="./stylesheets/pages/login-signup.css" rel="stylesheet" />
```
Comment (krishanu2911) [inline index.html]: just a small suggestion that you can move the stylesheet in one main CSS file the you only need to import only one CSS file in the header file

**Review by krishanu2911 — APPROVED** [review]
Well written code and very nice design just added a short suggestion.

**Review by aishwarya-bm — APPROVED** [review]
Looks good and clean, great usage of utils. 
Same suggestion as Krishanu mentioned

File: index.html (hunk 1 above)
Comment (vanshitalavange) [reply index.html]: changes done, thanks

**Pull request merged**